### PR TITLE
Added async declaration

### PR DESCRIPTION
**Pull request by keenondrums**
randomBytes method [is blocking](https://nodejs.org/api/crypto.html#crypto_crypto_randombytes_size_callback). In certain situations it's better to have asynchronous api

### CODE REVIEW

**Comment by sindresorhus** [comment]
I'm aware of how it works. I intentionally didn't bother with async as I couldn't think of any use-case when it would be useful:

> The only time when generating the random bytes may conceivably block for a longer period of time is right after boot, when the whole system is still low on entropy.

This means the only time it will block just a little bit is right after boot. So I'm curious what use-case you have that requires this?

**Comment by keenondrums** [comment]
There's not a particular case. It just bothers me to have a potentially blocking request in my production environment. What if spawn my docker containers with the app on demand? Will it block then? It just doesn't cost much to add async support and node8 will become LTS soon, so we're safe to use async functions.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "crypto-random-string",
-  "version": "1.0.0",
+  "version": "2.0.0",
```
Comment (sindresorhus) [inline package.json]: Don't bump version in a pull request. It's up to the maintainers when, how, and what version to bump.

**Comment by sindresorhus** [comment]
> What if spawn my docker containers with the app on demand?

But you spawn Docker containers though, not booting up machines, right? So they will already be seeded when you spawn the Docker containers.

I'm not saying I'm against this, I just want to be able to clearly document in the readme when you would want to use such async API. Most would never need it.

**Comment by keenondrums** [comment]
I would always use async if I need a random string on the fly. I'm kind of afraid of human-related factor here. So they say it blocks only on the start up, but what if there are other cases they did not count for. If there's a small chance of me stumbling upon this issue in production I'd definitely go async then. So from my point of view one should always use async when generating random string on the fly and can use sync for on time operations like populating dev db.

**Comment by sindresorhus** [comment]
I appreciate the PR, but I'm going to pass on this since I don't see any proven benefit. I usually always use async whenever possible too, but in this specific case it would IMHO just add needless overhead, both in execution speed and documentation.

> Premature optimization is the root of all evil

**Pull request closed without merging**